src/vistir/contextmanagers.py

```diff
@@ -345,7 +344,8 @@ def open_file(
             else:
                 session = Session()
         if session is None:
-            with closing(urlopen(link)) as f:
+            import urllib.request
```

Comment:
This style of import is problematic for pipenv to rewrite -- I converted `requirementslib` off this style imports for that reason, can you do `from urllib import request`?  Also why not have it at the top of the file?

Reply:
may not matter for this though since its a python builtin library.

Reply:
I fixed this as requested.